### DISCUSSION TITLE
Disable seek

### DISCUSSION THREAD

**Discussion by deepaktammali** [muxinc/media-chrome#668] [Q&A] [answered]
Hi There,

I have a use case where I need to disable video/audio seeking.
Is there a way we can achieve that in Media Chrome?

Thanks for the great library.

**Comment by cjpillsbury** [comment, marked as answer]
Hey @deepaktammali,
Just to clarify, if you're looking to disable any/all _**programmatic**_ seeking, Media Chrome does not provide any infrastructure to disable that.

If you're looking to just remove any user- or viewer-based seeking, you can do this with Media Chrome by simply:
1. not including any of our seek controls (e.g. `<media-seek-forward-button>`, `<media-time-range>`)
2. disabling seek hotkeys (`<media-controller hotkeys="noarrowleft noarrowright">`)

Let me know if that answers your question, or if I'm misunderstanding. Thanks for the Media Chrome love!

**Reply by deepaktammali** [reply]
Thanks for the response.

**Comment by cjpillsbury** [comment]
Going to mark this as closed due to inactivity @deepaktammali. If my prior response answered your question, it would be great if you can let other folks know by marking it as an answer. This helps others in the community who are asking similar questions.
